packages/@coorpacademy-components/src/atom/chips/index.js

```diff
@@ -1,3 +1,5 @@
+// This component is a specific version of the generic Chip component, customized for use in CoorpManager.
```

Comment:
```suggestion
// This component is a simpler version of the generic Chip component, in use in CoorpManager.
```
(technically it's not customized; as it's the original implementation, not coming from the Chip and tuned for the usage)